pipeline_dp/combiners.py

```diff
@@ -171,29 +199,61 @@ def merge_accumulators(self, accum1: AccumulatorType,
         count2, sum2 = accum2
         return count1 + count2, sum1 + sum2
 
-    def compute_metrics(self, accum: AccumulatorType) -> namedtuple:
+    def compute_metrics(self, accum: AccumulatorType) -> dict:
         total_count, total_sum = accum
         noisy_count, noisy_sum, noisy_mean = dp_computations.compute_dp_mean(
             total_count, total_sum, self._params.mean_var_params)
-        return MeanTuple(count=noisy_count, sum=noisy_sum, mean=noisy_mean)
+        mean_dict = {}
```

Comment:
Nit:
mean_dict = {'mean': noisy_mean}
if 'count' in self._metrics_to_compute:
    mean_dict['coumt'] = 
if 'sum' in...
   ...